src/code/z_debug.c

```diff
@@ -291,7 +293,7 @@ void Debug_DrawText(GraphicsContext* gfxCtx) {
         DebugCamera_DrawScreenText(&printer);
     }
 
-    if (gRegEditor->regPage != 0) {
+    if (OOT_DEBUG && gRegEditor->regPage != 0) {
```

Comment:
`#if` since `Regs_DrawEditor` was completely removed?

Reply:
yeah, this is also what was causing a match issue in debug. forgot to check